### PR TITLE
Fixed indicator types import/export for DTS emitter.

### PR DESCRIPTION
**Pull request by bre1470**

### CODE REVIEW

**Review by TorsteinHonsi — APPROVED** [review]
Looks good to me! @pawelfus can you have a look too?

File: ts/Stock/Indicators/VBP/VBPIndicator.ts [hunk 1]
```diff
@@ -34,11 +41,8 @@ const {
     noop
 } = H;
 import SeriesRegistry from '../../../Core/Series/SeriesRegistry.js';
-const {
-    seriesTypes: {
-        sma: SMAIndicator
-    }
-} = SeriesRegistry;
+const SMAIndicator: typeof SMAIndicatorType = SeriesRegistry.seriesTypes.sma;
+const columnPrototype = SeriesRegistry.seriesTypes.column.prototype;
```
Comment (pawelfus) [inline ts/Stock/Indicators/VBP/VBPIndicator.ts:45]: Shouldn't be:
```
import type ColumnSeriesType from '../../../Series/Column/ColumnSeries';
....
const columnProto: ColumnSeriesType = SeriesRegistry.seriesTypes.column.prototype;
```
Just like in other indicators (AOIndicator, MACDIndicator). 

Or maybe the other way around. `const columnPrototype = SeriesRegistry.seriesTypes.column.prototype;` is perfectly fine, and we can simply replace `const SMAIndicator: typeof SMAIndicatorType = SeriesRegistry.seriesTypes.sma;` with `const SMAIndicator = SeriesRegistry.seriesTypes.sma;`? Tested on RSIIndicator, seems to be fine

**Review by pawelfus — COMMENTED** [review]
Looks good! Just one question before merge (see below)

**Review by TorsteinHonsi — APPROVED** [review]
Thanks!

File: ts/Stock/Indicators/VBP/VBPIndicator.ts (hunk 1 above)
Comment (bre1470) [reply ts/Stock/Indicators/VBP/VBPIndicator.ts:45]: Right, I was thinking about that too and then decided to be rather consistent than mix deconstruction and regular assignment. In the produced codes deconstructions become regular assignments, so we do not produce overhead. Also some changes are temporary until we can produce native declarations and then have the assembler replaced to use regular named imports everywhere and remove all the import const and namespaces.

Comment (pawelfus) [reply ts/Stock/Indicators/VBP/VBPIndicator.ts:45]: Okay, thanks!

**Pull request merged**